common/beerocks/bwl/nl80211/ap_wlan_hal_nl80211.cpp

```diff
@@ -113,9 +113,9 @@ static uint8_t wpa_bw_to_beerocks_bw(const std::string &chan_width)
 // NOTE: Since *base_wlan_hal_nl80211* inherits *base_wlan_hal* virtually, we
 //       need to explicitly call it's from any deriving class
 ap_wlan_hal_nl80211::ap_wlan_hal_nl80211(const std::string &iface_name, hal_event_cb_t callback,
-                                         hal_conf_t hal_conf)
-    : base_wlan_hal(bwl::HALType::AccessPoint, iface_name, IfaceType::Intel, callback, hal_conf),
-      base_wlan_hal_nl80211(bwl::HALType::AccessPoint, iface_name, callback, BUFFER_SIZE)
+                                         const hal_conf_t &hal_conf)
+    : base_wlan_hal(bwl::HALType::AccessPoint, iface_name, IfaceType::Intel, callback),
```

Comment:
`base_wlan_hal ctor` also accept `hal_conf` as one of arguments:
```
    base_wlan_hal(HALType type, std::string iface_name, IfaceType iface_type,
                  hal_event_cb_t callback, hal_conf_t hal_conf = {});
```
So pass `hal_conf`into `base_wlan_hal` constructor too

Reply:
Fixed